### PR TITLE
[RM Ch05] Align with RM 2.4 changes

### LINKED ISSUE #2368: [RM Ch05] Modify 5.1 and 5.2 around Workload Profiles and Workload Flavours
Rewrite 5.1 and 5.2

5.3 and 5.4 content is better aligned with RM 4.2 -- only delete once RM content updated

5.1 Workload Profiles

5.2 Workload Flavours

5.2.1 Predefined Flavours

5.2.2 Parameterised Flavours

++++++++++++++++++++++++++++++++

Reference Issue #2354

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #2368

### CODE REVIEW

**Comment by pgoyal01** [comment]
Just started to make changes. Will add reviewers once it is complete.

File: doc/ref_model/chapters/chapter05.md [hunk 1]
```diff
@@ -123,11 +132,16 @@ This section details the services that may be made available to workloads by the
...
+|-----------------------|---------------|
+| Database as a service | Cassandra     |
+| Queue                 | Rabbit MQ     |
+| LB and HA Proxy       | NGINX, Istio  |
+| Security & Compliance | Calico        |
+| Monitoring            | Prometheus    |
+| Logging and Analysis  | ELK<sup*></sup> Stack |
+> <sup*></sup> ELK: Elasticsearch, Logstash, and Kibana
+
 <p align="center"><b>Table 5-7a:</b> Service examples.</p>
```
Comment (petorre) [inline doc/ref_model/chapters/chapter05.md:145]: "Service examples." -> "Platform Service Examples"?

File: doc/ref_model/chapters/chapter05.md [hunk 2]
```diff
@@ -123,11 +132,16 @@ This section details the services that may be made available to workloads by the
...
-| Minimum requirements | Example                                    |
-|----------------------|--------------------------------------------|
-|Database as a service | Cassandra                                  |
-|Queue                 | Rabbit MQ                                  |
-|LB and HA Proxy       |                                            |
+| Minimum requirements  | Example       |
+|-----------------------|---------------|
+| Database as a service | Cassandra     |
+| Queue                 | Rabbit MQ     |
+| LB and HA Proxy       | NGINX, Istio  |
```
Comment (petorre) [inline doc/ref_model/chapters/chapter05.md:144]: "Istio" to own line "Service Mesh"?

File: doc/ref_model/chapters/chapter05.md [hunk 3]
```diff
@@ -214,50 +236,18 @@ This section will detail Cloud Infrastructure Software Profiles and associated c
...
 
-This chapter defines a simplified host, host profile and related capabilities model associated with each of the different Cloud Infrastructure Hardware Profile and related capabilities; some of these profiles and capability parameters are shown in **Figure 5-4**.
+This chapter defines a simplified host, profile and related capabilities model associated with each of the different Cloud Infrastructure Hardware Profile and related capabilities; the [two profiles](./chapter02.md#241-node-profiles-top-level-partitions) (aka host profiles, node profiles, hardware profiles) and some of their associated capabilities are shown in **Figure 5-4**.
 
-<p align="center"><img src="../figures/RM_chap5_fig_5_4_HW_profile.png" alt="ref_hw_profiles" title="Reference HW Profiles" width="100%"/></p>
+<p align="center"><img src="../figures/RM-ch05-hw-profile.png" alt="ref_hw_profiles" title="Reference HW Profiles" width="80%"/></p>
```
Comment (petorre) [inline doc/ref_model/chapters/chapter05.md]: Doesn't show Figure 5-4 before changing URL.

File: doc/ref_model/chapters/chapter05.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter05.md:150]: @petorre Thanks. Done

File: doc/ref_model/chapters/chapter05.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter05.md:144]: @petorre Thanks. Done

File: doc/ref_model/chapters/chapter05.md (hunk 3 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter05.md]: @petorre Can you check again. It shows up for me.

**Comment by pgoyal01** [comment]
@walterkozlowski @petorre @karinesevilla @CsatariGergely @rgstori Ready for review.

Deleted some content including Figures that were not adding any value and in any case would require changes.

File: doc/ref_model/chapters/chapter05.md (hunk 3 above)
Comment (petorre) [reply doc/ref_model/chapters/chapter05.md]: @pgoyal01 : Yes, it does - I see it when I go to your patch. Somehow in PR "File changes" "Display rich diff" it doesn't show for me. Sorry.

**Review by petorre — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**